content/en/infrastructure/containers/configuration.md

```diff
@@ -195,3 +310,6 @@ Set the environment variable on both the Process Agent and Cluster Agent contain
 
 [1]: https://app.datadoghq.com/containers
 [2]: /infrastructure/containers
+[3]: https://github.com/DataDog/helm-charts/blob/main/charts/datadog/values.yaml#L651 
+[4]: /containers/amazon_ecs/?tab=awscli#setup
```

Comment:
```suggestion
```
I think these are the links that you'll have to move inside tabs, so you should be able to remove them from the bottom of the file.